client/flutter/assets/content_bundles/symptom_checker.en.yaml

```diff
@@ -142,3 +175,280 @@ contents:
+      severity: some_symptoms
+      display_condition: >-
+        wasDisplayed('q2_symptoms') && selectionCount('q2_symptoms') > 0 && !(has('fever') && has('dry_cough')) &&
+        (has('contact') || has('ltc_hcf')) &&
+        (has('over60') || has('preexisting_cond'))
+      title: Your symptoms might be caused by COVID-19, or could be caused by something else. Monitor your symptoms carefully.
+      cards:
+        - nonemergency_call
+        - testing
+        - quarantine
+        - wear_a_mask_if_coughing
+        - cough_tissue
+        - monitor_symptoms_getting_worse
+        - rest
+        - protect_others
+        - local_guidance
+    -
+      id: some_symptoms_nocall_quarantine
+      severity: some_symptoms
+      display_condition: >-
+        wasDisplayed('q2_symptoms') && selectionCount('q2_symptoms') > 0 && !(has('fever') && has('dry_cough')) &&
+        (has('contact') || has('ltc_hcf')) &&
+        has('no_preexisting_cond')
+      title: Your symptoms might be caused by COVID-19, or could be caused by something else. Monitor your symptoms carefully.
+      cards:
+        - testing
+        - quarantine
+        - wear_a_mask_if_coughing
+        - cough_tissue
+        - monitor_symptoms_getting_worse
+        - rest
+        - protect_others
+        - call_if_concerned
+    -
+      id: some_symptoms_call_stayhome
+      severity: some_symptoms
+      display_condition: >-
+        wasDisplayed('q2_symptoms') && selectionCount('q2_symptoms') > 0 && !(has('fever') && has('dry_cough')) &&
+        has('no_contact') &&
+        has('no_ltc_hcf') &&
+        (has('preexisting_cond') || has('over60'))
+      title: Your symptoms might be caused by COVID-19, or could be caused by something else. Monitor your symptoms carefully.
+      cards:
+        - nonemergency_call
+        - testing
+        - stay_home_until_better
+        - wear_a_mask_if_coughing
+        - cough_tissue
+        - monitor_symptoms_getting_worse
+        - rest
+        - protect_others
+        - local_guidance
+    -
+      id: some_symptoms_nocall_stayhome
+      severity: some_symptoms
+      display_condition: >-
+        wasDisplayed('q2_symptoms') && selectionCount('q2_symptoms') > 0 && !(has('fever') && has('dry_cough')) &&
+        has('no_contact') &&
+        has('no_ltc_hcf') &&
+        has('no_preexisting_cond')
+      title: Your symptoms might be caused by COVID-19, or could be caused by something else. Monitor your symptoms carefully.
+      cards:
+        - testing
+        - stay_home_until_better
+        - wear_a_mask_if_coughing
+        - cough_tissue
+        - monitor_symptoms_getting_worse
+        - rest
+        - protect_others
+        - call_if_concerned
+    -
+      id: no_symptoms_covid19contact
+      severity: none
+      title: You do not have symptoms consistent with COVID-19.
+      display_condition: >-
+        wasDisplayed('q2_symptoms') && selectionCount('q2_symptoms') == 0 &&
```

Comment:
In general, it feels like `wasDisplayed` is being used for non-null safety as opposed to something meaningful in the logic language. I would recommend (as part of this PR or a separate one) refactoring our evaluator so that we don't need to include `wasDisplayed` if we don't _need_ to know whether a question was displayed. I would think `selectionCount` should just be `0` for questions that haven't been asked, and answers on questions that haven't been asked should just evaluate to `false`. Let me know what you think

Reply:
Changing partially, as discussed in Slack.